src/styles/blocks/header.scss

```diff
@@ -0,0 +1,13 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+
+  
+}
+
+.header__logo {
```

Comment:
[Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) the 10th point